tools/test.py

```diff
@@ -28,6 +28,17 @@ def parse_args():
     parser.add_argument('config', help='test config file path')
     parser.add_argument('checkpoint', help='checkpoint file')
     parser.add_argument('--out', help='output result file')
+    out_options = ['class_scores', 'pred_score', 'pred_label', 'pred_class']
```

Comment:
'class_scores' and 'pred_score' may be confused,  may be “scores" and "pred_prob"

the 'scores' need a brief explainment in docstring.

Reply:
You're right, but more docstring may be too verbose, I think we can put these docs into our tools tutorials.